src/ValueResolver/NodeExpressionResolver.php

```diff
@@ -256,7 +255,12 @@ protected function resolveExprConstFetch(Expr\ConstFetch $node)
 
     protected function resolveExprClassConstFetch(Expr\ClassConstFetch $node)
     {
-        $refClass     = $this->fetchReflectionClass($node->class);
+        $classToReflect = $node->class;
```

Comment:
I'm  not sure why this issue isn't showing up for you, nor why I can't find links to individual Scrutinizer issues, but this is to address the Scrutinizer issue reported for this line that reads:

> <div><span style="color=white;background-color=grey">Bug</span> introduced 11 months ago by @lisachenko </div>
> It seems like <code style="color:red;">$node->class</code> can also be of type <code style="color:red;">object&lt;PhpParser\Node\Expr&gt;</code>; however, <code style="color:red;">Go\ParserReflection\Valu<span style="color=blue">...</span>:fetchReflectionClass()</code> does only seem to accept <code style="color:red;">object&lt;PhpParser\Node\Name&gt;</code>, maybe add an additional type check?